### ISSUE TITLE
Cluster parameters reorganization

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#91]
Originally reported of Google Code with ID 35.

```
Opening bug for tracking the cluster parameters (and
instance/node/hypervisor etc.) change.
```
Originally added on 2008-07-21 13:27:18 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was implemented and is available in 2.0b1.
```
Originally added on 2009-01-28 15:40:47 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
